simphony_ui/ui.py

```diff
@@ -240,6 +263,9 @@ def update_progress_bar(self, progress):
         """
         GUI.invoke_later(self.progress_dialog.update, progress)
 
+    def _logger_default(self):
+        return logging.getLogger(self.__class__.__name__)
```

Comment:
The names of loggers are hierarchical and convention is to use the (full, dotted) module names for the hierarchy.  If you really needed a class-specific logger here (which I don't think you do), the correct thing would be to do something like:
```
return logging.getLogger('{}.{}'.format(self.__class__.__module__.__name__, self.__class__.__name__))
```

It's in general much easier just to use a module-level global logger.